pkg/plugin/util/util.go

```diff
@@ -176,7 +180,19 @@ func UpdateSnapshotWithNewStorageClass(config *rest.Config, itemSnapshot *backup
 	}
 
 	// update the PVC storage class
-	old := *pvc.Spec.StorageClassName
+	var old string
+	_, ok := storageClassMapping[constants.EmptyStorageClass]
```

Comment:
I think this can be inside the if statement.  This is functionally fine, but it's clearer that it's only used inside the if statement if it's moved inside there.  I was looking for where ok's value would get used later on.

Reply:
Move line 184 inside if statement.